.github/workflows/formatting-linting.yml

```diff
@@ -19,19 +19,19 @@ jobs:
           node-version: 18
 
       - name: 'Install the dependencies'
-        run: 'yarn install'
+        run: 'yarn install --frozen-lockfile'
 
       - name: 'Check code formatting'
-        run: yarn prettier:check
+        run: 'yarn prettier:check'
```

Comment:
Don't we need use "run" any more?
like: `yarn run prettier:check`

Reply:
`run` can be omitted every since (see https://classic.yarnpkg.com/lang/en/docs/cli/run/#toc-yarn-run-script)